lib/BasicPlayer.tsx

```diff
@@ -42,18 +40,19 @@ interface BasicPlayerProps {
 
 export const BasicPlayer = forwardRef<PlayerNativeElement, BasicPlayerProps>(
   (
-    { hostname, vapixParams = {}, format, autoPlay = false, secure, className },
+    {
+      hostname,
+      vapixParams = {},
+      format = DEFAULT_FORMAT,
```

Comment:
`initialFormat`?

Reply:
The `BasicPlayer` doesn't allow you to change the format, therefore I used `format` here, while I used `initialFormat` on the `Player`, because there you can alter it through the controls (and a prop change will then not do anything).